### PR TITLE
Clarify which NTP package is used and when

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1637146

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL? Thanks!

File: day_two_guide/topics/ntp_synchronization.adoc [hunk 1]
```diff
@@ -13,6 +13,15 @@ keeping and time stamps, and is highly recommended for Kubernetes, which
 election, health checks for pods and some other issues, and helps prevent time
 skew problems.
 
+[NOTE]
+====
+The {product-title} installation playbooks install, enable, and configure the
+`ntp` package to provide NTP service unless explicitly disabled by setting
```
Comment (kalexand-rh) [inline day_two_guide/topics/ntp_synchronization.adoc]: I'd split this sentence. Maybe "...NTP service by default. To disable this behavior, set.."

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -110,7 +110,12 @@ a|This variable sets which INFO messages are logged to the `systemd-journald.ser
 For more information on debug log levels, see xref:../install_config/master_node_configuration.adoc#master-node-config-logging-levels[Configuring Logging Levels].
 
 |`openshift_clock_enabled`
-a| Whether to enable Network Time Protocol (NTP) on cluster nodes. `true` by default.
+a| Whether to enable Network Time Protocol (NTP) on cluster nodes.  `true` by
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: You have an extra space.
I'd say "The default value is `true`."

File: day_two_guide/topics/ntp_synchronization.adoc [hunk 3]
```diff
@@ -13,6 +13,15 @@ keeping and time stamps, and is highly recommended for Kubernetes, which
...
+`openshift_clock_enabled=false` in the inventory file. If a host has the
+`chrony` package already installed, it is configured to provide NTP service
+instead of using the `ntp` package.
+====
```
Comment (kalexand-rh) [inline day_two_guide/topics/ntp_synchronization.adoc]: What happens if you disable ntp but don't have chrony installed?

Comment (jboxman) [reply day_two_guide/topics/ntp_synchronization.adoc]: If `chrony` is not installed and NTP is not installed, the [playbook](https://github.com/openshift/openshift-ansible/blob/7d7afc9124dc7cfb84997014f50244dd0b0be546/playbooks/init/base_packages.yml#L16) installs `ntp`. If `ntp` is installed, but disabled (or `chrony`) the playbook [enables it](https://github.com/openshift/openshift-ansible/blob/7d7afc9124dc7cfb84997014f50244dd0b0be546/playbooks/init/base_packages.yml#L26).

**Pull request merged**